### PR TITLE
Added headers field in Entry

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | no
| Fixed tickets | #...
| License       | MIT

### CODE REVIEW

File: src/Wallabag/CoreBundle/Entity/Entry.php [hunk 1]
```diff
@@ -716,7 +725,7 @@ public function setPublishedAt(\Datetime $publishedAt)
     }
 
     /**
-     * @return string
+     * @return array
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Entity/Entry.php]: ?

File: src/Wallabag/CoreBundle/Entity/Entry.php [hunk 2]
```diff
@@ -734,4 +743,24 @@ public function setPublishedBy($publishedBy)
...
+    /**
+     * @return array
+     */
+    public function getHeaders()
+    {
+        return $this->headers;
+    }
+
+    /**
+     * @param string $headers
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Entity/Entry.php]: `array`

File: src/Wallabag/CoreBundle/Entity/Entry.php [hunk 3]
```diff
@@ -202,6 +202,15 @@ class Entry
     private $httpStatus;
 
     /**
+     * @var array
+     *
+     * @ORM\Column(name="headers", type="json_array", nullable=true)
```
Comment (tcitworld) [inline src/Wallabag/CoreBundle/Entity/Entry.php]: http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/types.html#json-array

Comment (tcitworld) [inline src/Wallabag/CoreBundle/Entity/Entry.php]: Use `json` instead.

File: src/Wallabag/CoreBundle/Entity/Entry.php (hunk 1 above)
Comment (nicosomb) [inline src/Wallabag/CoreBundle/Entity/Entry.php]: It's just a typo. `publishedBy` is an array too.

File: src/Wallabag/CoreBundle/Entity/Entry.php (hunk 3 above)
Comment (nicosomb) [inline src/Wallabag/CoreBundle/Entity/Entry.php]: OK, I need to change publishedBy too so.

File: src/Wallabag/CoreBundle/Entity/Entry.php (hunk 1 above)
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Entity/Entry.php]: Alright, then update `setPublishedBy` too

Comment (nicosomb) [inline src/Wallabag/CoreBundle/Entity/Entry.php]: Done.

**Pull request merged**